services/ui-src/src/components/menus/Sidebar.tsx

```diff
@@ -70,6 +70,14 @@ export const Sidebar = ({ isHidden }: SidebarProps) => {
             section={{ name: "placeholder", path: "/standard" }}
             level={1}
           />
+          <NavSection
+            key={"section name"}
```

Comment:
is this a placeholder?

Reply:
For now yes. I think both NavSection components will end up being removed once we bring in the routing. I have to check with Nick but I think it'll probably be one route mapping over whatever paths will be displayed.